tests/unit/test_unit_outdated.py

```diff
@@ -52,14 +55,14 @@ def test_pip_version_check(monkeypatch, stored_time, newver, check, warn):
                 "pip._vendor.six.moves",
                 "pip._vendor.requests.packages.urllib3.packages.six.moves",
             ]):
-        outdated.pip_version_check(session)
+        # TODO: Actually mock this well - @cooperlees
```

Comment:
Instead of a mock, you could just use our data fixture and create a local, on disk, ``--find-links`` path that contains a copy of pip. You'll probably need to mock ``get_installed_version`` so that it returns a hardcoded value that is less than whatever version you have hidden inside that ``--find-links`` path (and possible also one that is equal and greater than).

Alternatively, instead of passing in a ``session`` and an ``options`` and constructing the ``PackageFinder`` inside of this method, you could instead change the signature so that it accepts an already created ``PackageFinder``. Then you can just pass in a stub finder that returns whatever value you want.